src/science/landuse/landuse3.F90

```diff
@@ -3,7 +3,25 @@ MODULE landuse_variable
   IMPLICIT NONE
 
   SAVE
-
+!*# Overview of landuse3.F90
+!
+! These are three-type landuse-type variables:
+!
+! - `luc%var_x(mland,mvmax)`: variable indexed by "mland" and "mvmax" before landuse change
+! - `luc%var_y(mland,mvmax)`: variable indexed by "mland" and "mvmax" after land use change
+! - `lucmp%var( mp        )`: variable indexed by the patch number from 1 to mp
+!
+! \[$$\textbf{y} = \textbf{x} \cdot \textbf{T}\]
```

Comment:
```suggestion
The land use change for each variable is applied in the form of a transformation matrix `T` such that:
! \[$$\textbf{luc%var_y} = \textbf{luc%var_x} \cdot \textbf{T}\]
```
I am not certain what the equation is. This is a suggestion based on my understanding but feel free to adapt/reject if I got it wrong.